api/renter_test.go

```diff
@@ -101,8 +234,118 @@ func TestRenterDownloadError(t *testing.T) {
 	}
 }
 
-// TestRenterAsyncDownloadError tests that the /renter/asyncdownload route sets the download's error field if it fails.
-func TestRenterAsyncDownloadError(t *testing.T) {
+// TestValidDownloads tests valid and boundary parameter combinations.
+func TestValidDownloads(t *testing.T) {
```

Comment:
this test takes over a minute to run. Maybe drop some test cases or run them in parallel.